File: qt.rb
#!/usr/bin/env ruby
require 'Qt'

app = Qt::Application.new(ARGV)
# Qt.debug_level = Qt::DebugLevel::High

menu = Qt::Menu.new

click = menu.add_action("Clickme")
click.connect(SIGNAL :triggered) do
  puts "Clicked"
end
quit = Qt::Action.new("Quitme", menu) do
  connect(SIGNAL :triggered) { app.quit }
end
f = quit.font
f.set_bold(true)                # f.set_italic(true) 
quit.font = f
menu.add_action(quit)

menu.exec

File: qt.py
#!/usr/bin/env python3
from PyQt5.QtCore import (Qt, QPoint)
from PyQt5.QtGui import QCursor
from PyQt5.QtWidgets import (QApplication, QMenu, QAction)

def clicked():
    print("CLICKED")

if __name__ == '__main__':
    import sys

    app = QApplication(sys.argv)
    menu = QMenu()
    # must assign a variable, passing directly does not work
    # see https://stackoverflow.com/questions/57807036
    action = QAction("&Click me", triggered=clicked)
    menu.addAction(action)
    # menu.show()
    menu.exec_(QCursor().pos() + QPoint(10, 20))
    #sys.exit(app.exec_())
